app/src/main/java/io/zenandroid/onlinego/ui/screens/stats/StatsFragment.kt

```diff
@@ -1,26 +1,65 @@
 package io.zenandroid.onlinego.ui.screens.stats
 
+import android.graphics.Typeface.BOLD
+import android.graphics.Typeface.NORMAL
 import android.os.Bundle
-import androidx.fragment.app.Fragment
+import android.text.SpannableString
+import android.text.SpannableStringBuilder
+import android.text.Spanned.SPAN_EXCLUSIVE_EXCLUSIVE
+import android.text.style.AbsoluteSizeSpan
+import android.text.style.ForegroundColorSpan
+import android.text.style.RelativeSizeSpan
+import android.text.style.StyleSpan
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.load.resource.drawable.DrawableTransitionOptions
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.transition.DrawableCrossFadeFactory
+import com.github.mikephil.charting.charts.BarLineChartBase
+import com.github.mikephil.charting.components.AxisBase
+import com.github.mikephil.charting.components.XAxis
+import com.github.mikephil.charting.data.*
+import com.github.mikephil.charting.formatter.ValueFormatter
+import com.github.mikephil.charting.utils.EntryXComparator
 import io.zenandroid.onlinego.OnlineGoApplication
 import io.zenandroid.onlinego.R
+import io.zenandroid.onlinego.data.model.ogs.OGSPlayer
+import io.zenandroid.onlinego.gamelogic.Util
+import io.zenandroid.onlinego.utils.convertCountryCodeToEmojiFlag
+import io.zenandroid.onlinego.utils.egfToRank
+import io.zenandroid.onlinego.utils.formatRank
+import io.zenandroid.onlinego.utils.processGravatarURL
+import kotlinx.android.synthetic.main.view_player_profile.*
+import kotlinx.android.synthetic.main.view_stats.*
+import org.koin.android.ext.android.get
+import java.text.SimpleDateFormat
+import java.util.*
 
 class StatsFragment : Fragment(), StatsContract.View {
 
+    companion object {
+        fun createFragment(id: Long): StatsFragment = StatsFragment().apply {
+                playerID = id
+        }
+    }
+
     private lateinit var presenter: StatsContract.Presenter
     private var analytics = OnlineGoApplication.instance.analytics
+    private var playerID = Util.getCurrentUserId()!!
+    private var mostFacedOpponentGameRecords : List<Map<String, String>>? = null
+    private var highestWinGameRecord : List<Map<String, String>>? = null
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        return inflater.inflate(R.layout.fragment_stats, container, false)
+        return inflater.inflate(R.layout.view_stats, container, false)
```

Comment:
the modern way is to put `R.layout.view_stats` in the constructor of fragment in the first line of the class like this: `class StatsFragment : Fragment(R.layout.view_stats), StatsContract.View {`

Reply:
Fixed in https://github.com/acristescu/OnlineGo/commit/1c57f6a852393483b9a7b3936e3048083c23aed1